twisted/topfiles/8577.bugfix

```diff
@@ -1 +1,5 @@
 trial can now run some tests on Windows with Python 3
+
```

Comment:
I have no idea how the topfiles ought to work, but maybe it'd be nice to just put the commit message:

`Remove "python" before invoking tests via "tox -e py35-tests"`

...and if it *is* nice, we should consider automating this.